### DISCUSSION TITLE
kvs webrtc with storing the stream

### DISCUSSION THREAD

**Discussion by aniket0527** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1929] [General]
there is raspberry pi which is live streaming video(uni direction) and audio(bi direction) to client app and also I want to store that. so it it possible with kvs webrtc? because i already try it but i cant able to store and there is new release like ingestion and storage media but in that client app (viewer) unable to join and master getting stucked "PROFILE initSignal(): [Signling connect client] 2147ms"

**Comment by disa6302** [comment]
Hello, 

Can you clarify your use case? What is it that you want to store? What SDKs are you using for master and viewer?
